src/controls/extras/tree.coffee

```diff
@@ -170,6 +170,9 @@ class Tree extends Typed
           setTimeout(arguments.callee, 0)
 
     timer = false
+    bandsCountId = "bands-count-" + $('.bands-count').length
```

Comment:
Sorry, I should have make it clearer - I meant the grand parent of the jstree container.
For example, this is a granule retrieval of an ESI capable collection:
http://edsc.dev/data/configure?p=C1236303848-NSIDC_ECS&tl=1473013240!4!!&q=smap&ok=smap&ff=Subsetting+Services&sgd=G1373821066-NSIDC_ECS&back=%2Fsearch%2Fgranules

The ID of the container of the jstree, the count and the filter box is ```SMAPL1L2-bandSubsetOptions-47-element``` and the grand parent ID is ```SMAPL1L2-bandSubsetOptions-47```.  

If you ```Add access method``` and choose ```SPL1CTB.3 ESI Service``` again, the ID of the grand parent will be different ```SMAPL1L2-bandSubsetOptions-95```.